analogway_livecore.js

```diff
@@ -1,743 +1,564 @@
-				},{
-					type: 'dropdown',
-					label: 'Screen 7',
-					id: '6',
-					default: '0',
-					choices: [ { id: '0', label: 'No change' }, { id: '1', label: 'Add to selection' }, { id: '2', label: 'Remove from selection' }]
-				},{
-					type: 'dropdown',
-					label: 'Screen 8',
-					id: '7',
-					default: '0',
-					choices: [ { id: '0', label: 'No change' }, { id: '1', label: 'Add to selection' }, { id: '2', label: 'Remove from selection' }]
-			}]},
-			'loadconfidence': {
-				label: 'Load confidence Memory',
-				options: [{
-					type: 'textinput',
-					label: 'Memory',
-					id: 'memory',
-					default: '',
-					tooltip: 'Enter the number of the memory you want to load from 1 to 16',
-					regex: '/^0*([1-9]|1[0-6])$/'
-				},{
-					type: 'textinput',
-					label: 'Destination screen',
-					id: 'destscreen',
-					default: '1',
-					tooltip: 'Enter the number of the screen where the memory schould be loaded to (1 to 8)',
-					regex: '/^0*[1-8]$/'
-			}]},
-			'activateconfidence': {
-				label: 'Switch confidence mode',
-				options: [{
-				type: 'textinput',
-				label: 'Screen',
-				id: 'destscreen',
-				default: '1',
-				tooltip: 'Enter the number of the screen where confidence mode should be switched (1 to 8)',
-				regex: '/^0*[1-8]$/'
-			},{
-				type: 'dropdown',
-				label: 'Mode',
-				id: 'mode',
-				default: '0',
-				tooltip: 'Select wether confidence mode should be switched on or off at selected screen',
-				choices: [ { id: '0', label: 'Off' }, { id: '1', label: 'On' }]
-			}]},
-			'switchplug': {
-				label: 'Switch input plug',
-				tooltip: 'Note that not all inputs may be available at your system and not all plugs are available at any input.',
-				options: [{
-				type: 'textinput',
-				label: 'Input',
-				id: 'input',
-				default: '1',
-				tooltip: 'Enter the number of the input where you want to switch the input plug (1 to 12 for inputs of master device and 13 to 24 for inputs of slave device).',
-				regex: '/^0*([1-9]|1[0-9]|2[0-4])$/'
-			},{
-				type: 'dropdown',
-				label: 'Plug',
-				id: 'plug',
-				default: '0',
-				tooltip: 'Select the plug to use for that input',
-				choices: [ { id: '0', label: 'Analog HD15 connector' }, { id: '1', label: 'Analog DVI-A connector' }, { id: '2', label: 'DVI' }, { id: '3', label: 'SDI' }, { id: '4', label: 'HDMI' }, { id: '5', label: 'DisplayPort' } ]
-			}]},
-			'sendcustomcommand': {
-				label: 'Send custom command',
-				options: [{
-					type: 'textinput',
-					label: 'Command',
-					id: 'command',
-					default: '',
-					tooltip: "Enter any command you like in plain ASCII. Beware of correct syntax, you mustn't enter the linebreak at the end of the command.",
-					regex: '/^[\\w,]+$/i'
-			}]}
-	});
-}
+	feedback(feedback) {
+		if (feedback.type === 'input_active') {
+			if (tallyPGM.includes(feedback.options.source)) {
```

Comment:
See suggestion above
```suggestion
			if (tallyPGM[feedback.options.source]) {
```